### PR TITLE
Doc: remove obsolete section on reading BTD data

### PR DESCRIPTION
**Pull request by RemiLehe**
Since the only option for BTD data now is to use the regular plotfile or openPMD format, this section on the obsolete backtransformed format is no longer need.

### CODE REVIEW

**Review by RevathiJambunathan — CHANGES_REQUESTED** [review]
Thank you @RemiLehe for this PR.
Please can you also remove these files as they were added to postprocess the customized BTD output

https://github.com/ECP-WarpX/WarpX/blob/development/Tools/PostProcessing/read_lab_particles.py

https://github.com/ECP-WarpX/WarpX/blob/436934f18119543604a9c33fcb7eefcc162b4be3/Tools/PostProcessing/read_raw_data.py#L54 and this function

File: Tools/PostProcessing/read_raw_data.py [hunk 1]
```diff
@@ -294,7 +217,6 @@ def _read_buffer(snapshot, header_fn, _component_names):
 
     boxes, file_names, offsets, header = _read_header(header_fn)
 
-    ng = header.nghost
```
Comment (RemiLehe) [inline Tools/PostProcessing/read_raw_data.py:297]: This was unused.

File: Tools/PostProcessing/read_raw_data.py [hunk 2]
```diff
@@ -262,7 +186,6 @@ def _read_field(raw_file, field_name):
     header_file = raw_file + field_name + "_H"
     boxes, file_names, offsets, header = _read_header(header_file)
 
-    ng = header.nghost
```
Comment (RemiLehe) [inline Tools/PostProcessing/read_raw_data.py:265]: This was unused.

**Review by RevathiJambunathan — APPROVED** [review]
Thank you @RemiLehe

**Review by EZoni — APPROVED** [review]

**Pull request merged**